From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Williamson <chris@chrisaw.io>
Date: Wed, 05 Feb 2026 00:00:00 +0000
Subject: [PATCH 2/3] Use GrapheneOS app names in GSI Android.bp

GrapheneOS uses different module names for some apps:
- Camera (instead of Camera2)
- Messaging (instead of messaging)
- TrichromeChrome (instead of Browser2) - Vanadium browser

This patch updates the GSI build configuration to use the
GrapheneOS-provided equivalents.

Change-Id: I0000000000000000000000000000000000000001
---
 target/product/gsi/Android.bp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/target/product/gsi/Android.bp b/target/product/gsi/Android.bp
--- a/target/product/gsi/Android.bp
+++ b/target/product/gsi/Android.bp
@@ -120,11 +120,11 @@ android_filesystem_defaults {
         ///////////////////////////////////////////
         // gsi_product
         ///////////////////////////////////////////
-        "Browser2",
-        "Camera2",
+        "TrichromeChrome",
+        "Camera",
         "Dialer",
         "LatinIME",
-        "messaging",
+        "Messaging",
         "apns-full-conf.xml",
         "frameworks-base-overlays",
     ],
-- 
2.34.1

